avm/res/document-db/database-account/tests/e2e/role/main.test.bicep

```diff
@@ -19,23 +19,23 @@ param namePrefix string = '#_namePrefix_#'
 
 // Pipeline is selecting random regions which dont support all cosmos features and have constraints when creating new cosmos
 #disable-next-line no-hardcoded-location
-var enforcedLocation = 'eastus2'
+var resourceLocation = 'eastus2'
```

Comment:
Is the intention to enforce a location just for this test? when using an enforced location please keep the `enforcedLocation` convention

Reply:
I'll make this change